NYPLCardCreator/Classes/AddressViewController.swift

```diff
@@ -115,8 +95,9 @@ final class AddressViewController: FormTableViewController {
     self.cityCell.textField.keyboardType = .Alphabet
     self.cityCell.textField.autocapitalizationType = .Words
     
-    self.regionCell.textField.inputView = self.regionPickerView
-    self.regionCell.textField.inputAccessoryView = self.returnToolbar()
+    self.regionCell.textField.keyboardType = .Alphabet
+    self.regionCell.textField.autocorrectionType = .No
```

Comment:
I was thinking it'd be more of a annoyance for users trying to type two letter abbreviations, but I've been out-voted on this one!